### PR TITLE
Add visible aspects of Wild Life of Our Homes cnxn

### PR DESCRIPTION
**Pull request by madprime**

### CODE REVIEW

File: open_humans/templates/member/my-member-research-data.html [hunk 1]
```diff
@@ -161,7 +161,6 @@ <h4 class="panel-title">Studies</h4>
       {% with user_data=user.pgp %}
          {% include 'partials/connection-study.html' %}
       {% endwith %}
-    </div>
```
Comment (beaugunderson) [inline open_humans/templates/member/my-member-research-data.html]: @madprime looks like this was removed in error?

**Comment by beaugunderson** [comment]
@madprime looks good aside from the one comment :)

**Comment by madprime** [comment]
Removing WIP and assigning @beaugunderson -- I think this is ready for review and Wild Life of Our Homes should be going to production. We anticipate sending email invitations soon.

**Comment by beaugunderson** [comment]
@madprime looks good to me! :+1:

**Pull request merged**